src/app/components/menubar/menubar.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component } from '@angular/core';
+import { BrowserWindow } from 'electron';
+
+@Component({
+  selector: 'app-menubar',
+  templateUrl: './menubar.component.html',
+  styleUrls: ['./menubar.component.scss']
+})
+export class MenubarComponent {
+  isFullscreen: boolean;
+  window;
```

Comment:
i think u should give this the type "any" (i.e., window: any) to maintain consistency with the rest of our code